src/plugins/pcr_primer_design_for_dna_assembly/src/options_panel/ResultTable.h

```diff
@@ -36,26 +36,32 @@ struct ResultTableData {
     AnnotatedDNAView *associatedView = nullptr;
     AnnotationGroup *associatedGroup = nullptr;
     QList<U2Region> currentProducts;
+    QPair<int, int> primerLengths;
```

Comment:
Please document what is the first/second in the pair here and below (getSelectedFragment).
I would recommend not using Pair in the public API, because it is very unreadable (requires a lot of comments)